nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -58,6 +58,11 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - `libass` now uses the native CoreText backend on Darwin, which may fix subtitle rendering issues with `mpv`, `ffmpeg`, etc.
 
+- [`lib.customisation.extendMkDerivation`](https://nixos.org/manual/nixpkgs/unstable#function-library-lib.customisation.extendMkDerivation) is introduced to define a build helper supporting [fixed-point arguments](https://nixos.org/manual/nixpkgs/unstable#chap-build-helpers-finalAttrs) (`(finalAttrs: { ... })`) based on an existing one with such support.
+  It can convert existing build helper definitions to take fixed-point arguments with little expression changes.
+
+  See https://nixos.org/manual/nixpkgs/unstable#sec-build-helper-extendMkDerivation for detail.
```

Comment:
```suggestion
  See [](#sec-build-helper-extendMkDerivation) for details.
```

Or whatever the implicit link text syntax is here

Reply:
The release note is inside the NixOS manual. Is it even possible for it to interlink between the Nixpkgs manual this way?

Reply:
Alright, see above. Sorry for the noise.